riscv/isa_parser.cc

```diff
@@ -240,6 +240,9 @@ isa_parser_t::isa_parser_t(const char* str, const char *priv)
       extension_table[EXT_ZVFBFWMA] = true;
     } else if (ext_str == "sstc") {
         extension_table[EXT_SSTC] = true;
+    }
+    else if (ext_str == "smcntrpmf") {
```

Comment:
It's  better to add "else if" to last line.

Reply:
Done.